File: serverTCC/serverTCC/Models/Usuario.cs
using Microsoft.AspNetCore.Identity.EntityFrameworkCore;

namespace serverTCC.Models
{
    public class Usuario : IdentityUser
    {
        public string Nome { get; set; }
        public int PerfilId { get; set; }
        public Perfil Perfil { get; set; }
    }
}

File: client/www/js/models/usuario.js
function Usuario(data) {
    "use strict";
    if (!data) data = {};

    this.id = data.id;
    this.nome = data.nome;
    this.perfilId = data.perfilId;
    if (data.perfil)
        this.perfil = new Perfil(data.perfil);
}

localEntities.register(Usuario, {
    id: {
        type: 'number',
        pk: true
    },
    nome: {
        type: 'string'
    },
    perfilId: {
        type: 'number',
        fk: true
    },
    perfil: {
        type: 'Perfil',
        relationKey: 'perfilId'
    },
});